english/sleep/hungry/hungry.md

```diff
@@ -0,0 +1,3 @@
+You can't believe yourself.You are like,"did I just acquire some kind of superpowers or something??How is all this happening?".But now you are hungry, your stomach starts to growl.so, you put aside all those thoughts and start eating the chocolate furniture.Yum!Yum!Yum!!!when you are finished eating more than half of your furniture.You try to go back to sleep.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.